code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -246,23 +246,22 @@ There are important things regarding this file:
 		M.adjust_fire_stacks(fire_stacks)
 		M.IgniteMob()
 
-//Should do about 80 damage at 1 tile distance (adjacent), and 50 damage at 3 tiles distance.
-//Overall less damage than slugs in exchange for more damage at very close range and more embedding
-/obj/item/projectile/bullet/pellet/shotgun
+
+/obj/item/projectile/bullet/shotgun/buckshot
 	name = "shrapnel"
 	icon_state = "birdshot-1"
-	damage_types = list(BRUTE = 10)
-	pellets = 8
-	range_step = 1
-	spread_step = 10
-	knockback = 1
+	damage_types = list(BRUTE = 37)
+	armor_penetration = 40
```

Comment:
Yep. However, if you pair high damage with falloff and increase the damage even more as a compensation for being less useful at range, this leads to the ammo becoming a designated close range killer against people without armor. This is not good because it discourages people from not running around in full bulletproof all the time, but it also makes the ammo type rather useless against other combat-geared people, as these will have armor and will not let you get close if they can.

Anyways, people will adapt to this change pretty quickly without too much confusion, I hope. maybe I could add a note about the enhanced armor-piercing capabilities of futuristic buckshot, or just rename buckshot into flechette :P

Reply:
> this leads to the ammo becoming a designated close range killer against people without armor
This doesn't makes any sense at all - any ammo, bar rubbers, can become a `designated close range killer` if the target isn't wearing armor.
It's a ton of small pellets, it indeed shouldn't be effective against any kind of good armor.
```suggestion
	armor_penetration = 0
```

Reply:
This would make buckshot absolutely and utterly useless. And a weapon that Deals 100+ damage to An unarmored target on PB is much more dangerous than your Standard issue Rifle Dealing like 40 damage on PB. Test for yourself

Reply:
Pellets having that high armor penetration does not makes any sense at all — it indeed should be utterly useless against any armor.

Reply:
If it's useless against armor, it will only be used against unarmored people. If it is not more effective at killing unarmored people than other ammo Types of similar Rarity, it will not be used at all. I propose Making buckshot 60 damage projectiles with 0 AP and slug hat damage falloff at highest Range, while giving slugs 45 damage and 20 AP, so you have an actual choice bewegen AP and non AP ammo. But from the Perspective of Having interesting ammo, a close Range anti armor bullet still Seems like a better idea to me and it could be easily Renamed to flechette for a bit more Realism, which shouldnt Play a Big Role Here IMO. Only so far as to make people realize that ammo type X is supposed to be Good at Y, not because we need an accurate ballistics simulation to have fun.